daemon/config/config.go

```diff
@@ -20,26 +22,29 @@ import (
 )
 
 const (
-	// defaultMaxConcurrentDownloads is the default value for
+	// DefaultMaxConcurrentDownloads is the default value for
 	// maximum number of downloads that
 	// may take place at a time for each pull.
-	defaultMaxConcurrentDownloads = 3
-	// defaultMaxConcurrentUploads is the default value for
+	DefaultMaxConcurrentDownloads = 3
+	// DefaultMaxConcurrentUploads is the default value for
 	// maximum number of uploads that
 	// may take place at a time for each push.
-	defaultMaxConcurrentUploads = 5
-	// stockRuntimeName is the reserved name/alias used to represent the
+	DefaultMaxConcurrentUploads = 5
+	// StockRuntimeName is the reserved name/alias used to represent the
 	// OCI runtime being shipped with the docker daemon package.
-	stockRuntimeName = "runc"
+	StockRuntimeName = "runc"
 )
 
 const (
-	defaultNetworkMtu    = 1500
-	disableNetworkBridge = "none"
+	// DefaultNetworkMtu is the default value for network MTU
+	DefaultNetworkMtu = 1500
+	// DisableNetworkBridge is the default value of the option to disable network bridge
+	DisableNetworkBridge = "none"
 )
 
 const (
-	defaultShutdownTimeout = 15
+	// DefaultShutdownTimeout is the default shutdown timeout for the daemon
+	DefaultShutdownTimeout = 15
```

Comment:
This constant appears to be only used in one place. It would be better to keep it unexported and move it to where it is used.